src/components/PlayerView/index.tsx

```diff
@@ -49,7 +50,7 @@ export function PlayerView({
   fullscreenHandler,
   customMessageHandler,
   isFullscreenRequested = false,
-  scalingMode,
+  scalingMode = ScalingMode.Fit,
```

Comment:
The alternative is to default in the Kotlin code on null.
Chose to default in the TS code aligned with `isFullscreenRequest`.